pkg/apis/options/upstreams.go

```diff
@@ -49,12 +49,13 @@ type Upstream struct {
 	// Defaults to 1 second.
 	FlushInterval *time.Duration `json:"flushInterval,omitempty"`
 
-	// PassHostHeader determines whether the request host header should be proxied
-	// to the upstream server.
-	// Defaults to true.
-	PassHostHeader bool `json:"passHostHeader"`
-
-	// ProxyWebSockets enables proxying of websockets to upstream servers
-	// Defaults to true.
-	ProxyWebSockets bool `json:"proxyWebSockets"`
+	// SkipHostHeader determines whether the request host header should be proxied
+	// to the upstream server. The host header is passed by default.
+	// Defaults to false.
+	SkipHostHeader bool `json:"passHostHeader"`
+
+	// SkipWebSockets disables proxying of websockets to upstream servers.
+	// Websockets are proxied by default.
+	// Defaults to false.
+	SkipWebSockets bool `json:"proxyWebSockets"`
```

Comment:
Similar here for the word `Skip` -- maybe `EnableWebSockets`?

Reply:
Yeah equally good point, can either go down the `DisableWebSockets` road or the pointer to bool route, WDYT?